cli/command/service/update_test.go

```diff
@@ -1522,6 +1522,28 @@ func TestUpdateCaps(t *testing.T) {
 			expectedAdd:  []string{"CAP_AAA", "CAP_BBB", "CAP_CCC", "CAP_DDD"},
 			expectedDrop: []string{"CAP_WWW", "CAP_XXX", "CAP_YYY", "CAP_ZZZ"},
 		},
+		{
+			name:     "Reset capabilities",
+			flagAdd:  []string{"RESET"},
```

Comment:
Should we add also a test with `ALL`, `RESET` ?

Reply:
Added two tests that combine `ALL` and `RESET`